ApiReview/API.Polly.Extensions/NoDocs/Polly/TelemetryResilienceStrategyBuilderExtensions.cs

```diff
@@ -0,0 +1,12 @@
+// Assembly 'Polly.Extensions'
+
+using Microsoft.Extensions.Logging;
+using Polly.Extensions.Telemetry;
+
+namespace Polly;
+
+public static class TelemetryResilienceStrategyBuilderExtensions
+{
+    public static TBuilder EnableTelemetry<TBuilder>(this TBuilder builder, ILoggerFactory loggerFactory) where TBuilder : ResilienceStrategyBuilderBase;
```

Comment:
Since telemetry is enabled by default, I think it would be better to call this `ConfigureTelemetry` instead of `EnableTelemetry`. This way I would assume that telemetry is not enabled until I call this.